themes/index.js

```diff
@@ -1,4 +1,10 @@
 export const themes = {
+  dark_red: {
```

Comment:
Thanks for your contribution. I'm happy to merge this theme if you put the code under the `dark` theme 👍🏻.

Reply:
Thank you <3